.env.example

```diff
@@ -4,12 +4,25 @@ TRUST_PROXY=true
 AUTHGEAR_APP_ID=accounts
 AUTHGEAR_CLIENT_ID=portal
 AUTHGEAR_ENDPOINT=http://accounts.portal.localhost:3000
+
 DATABASE_SCHEMA=app
 DATABASE_URL=postgres://postgres:postgres@localhost:5432/postgres?sslmode=disable
+
+AUDIT_DATABASE_SCHEMA=app
+AUDIT_DATABASE_URL=postgres://postgres:postgres@localhost:5432/postgres?sslmode=disable
+
+REDIS_URL=redis://localhost
+
+ANALYTIC_REDIS_URL=redis://localhost
```

Comment:
Maybe we can use another db for ANALYTIC_REDIS_URL during development. e.g. `redis://localhost/1`